src/cmd/pkg/auth/auth.go

```diff
@@ -208,19 +194,43 @@ func (c *subcommand) newListCmd() *cobra.Command {
 	return cmd
 }
 
-func setDefaultOrg(store *token.Store, org string) error {
-	return errors.WithStack(store.Put(defaultOrgPath, token.Token{
-		Value: []byte(org),
-		// Expire the default org if not updated for 1 year
-		Expiry: time.Now().Add(time.Hour * 24 * 365),
-	}))
+type whoAmIResponse struct {
+	Email string `json:"email"`
 }
 
-// GetDefaultOrg returns the default org within the store
-func GetDefaultOrg(store *token.Store) (string, error) {
-	org, err := store.Get(defaultOrgPath)
-	if err != nil {
-		return "", errors.WithStack(err)
+func (c *subcommand) newWhoAmICmd() *cobra.Command {
+	authFlags := client.NewClientFlags()
+	cmd := &cobra.Command{
+		Use:     "whoami",
+		GroupID: groups.Commands.ID,
+		Short:   "Print the currently authenticated user.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			req, err := authFlags.NewRequest(http.MethodGet, "/auth/whoami", nil /* body */)
+			if err != nil {
+				return errors.WithStack(err)
+			}
+			client := http.Client{}
+			resp, err := client.Do(req)
+			if err != nil {
+				return errors.WithStack(err)
+			}
+			defer resp.Body.Close()
+			body, err := ioutil.ReadAll(resp.Body)
+			if err != nil {
+				return errors.WithStack(err)
+			}
+			if resp.StatusCode != http.StatusOK {
+				return errors.New(string(body))
```

Comment:
nit suggest prefixing error with the http status here, in case the body has nothing useful